src/app/api/auth/next-auth.ts

```diff
@@ -33,16 +43,27 @@ const nextAuth = NextAuth({
     },
   },
   providers: ENABLE_OAUTH_SSO
-    ? [
-        Auth0({
-          // Specify auth scope, at least include 'openid email'
-          // all scopes in Auth0 ref: https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes#standard-claims
-          authorization: { params: { scope: 'openid email profile' } },
-          clientId: AUTH0_CLIENT_ID,
-          clientSecret: AUTH0_CLIENT_SECRET,
-          issuer: AUTH0_ISSUER,
-        }),
-      ]
+    ? SSO_PROVIDER.split(',').map((provider) => {
```

Comment:
I think we can have multi providers, not only one. Is this possible?

Reply:
Did you mean, to change the param name from `SSO_PROVIDER` to `SSO_PROVIDERS`?

Yes, the current logic support mulit provides.
you can check my instance [auth link](https://lobe.xllb.cc/api/auth/signin?callbackUrl=https%3A%2F%2Flobe.xllb.cc%2Fsettings%2Fcommon).

Reply:
Although your suggestion is beneficial for the user config the SSO, I think the current solution is fine, because the user may want to control the SSO prviders display order.

If we are using `ENABLE_AUTH0` and `ENABLE_AZURE_AD` splitly, the env `ENABLE_OAUTH_SSO` may need to change.

Is that make sense?

Reply:
> because the user may want to control the SSO provider display order.

You hit me, I think `SSO_PROVIDERS` is better. 

Only one stuff left:  
```
SSO_PROVIDERS=["auth0","azure_ad"]
```
or 
```
SSO_PROVIDERS=auth0,azure_ad
```

which is better ?

Reply:
> which is better ?

I would like the `SSO_PROVIDERS=auth0,azure_ad` this one.